tensorflow/core/kernels/segment_reduction_ops.cc

```diff
@@ -332,7 +330,8 @@ class UnsortedSegmentSumOp : public UnsortedSegmentBaseOp<Device, T, Index> {
   explicit UnsortedSegmentSumOp(OpKernelConstruction* context)
       : UnsortedSegmentBaseOp<Device, T, Index>(
             context,
-           *(new functor::UnsortedSegmentSumFunctor<Device, T, Index>())) {}
+            sumFunctor) {}
+    functor::UnsortedSegmentSumFunctor<Device, T, Index> sumFunctor;
```

Comment:
@drpngx thanks for the comment. I'd be very grateful for some input from your side, how can I best reach you?